labs/lab3-paxos/src/dslabs/paxos/PaxosServer.java

```diff
@@ -39,7 +39,14 @@ public void init() {
        -----------------------------------------------------------------------*/
 
     /**
-     * Return the status of a given slot in the servers's local log.
+     * Return the status of a given slot in the server's local log. This
+     * includes acceptances on this node.
```

Comment:
I don't really like "this includes acceptances on this node." It's already implied by the inclusion of ACCEPTED as a possible status of a slot.